common/source/docs/common-auxiliary-functions.rst

```diff
@@ -0,0 +1,633 @@
++----------------------+----------------------------+----------+---------+---------+
+|        62            | Compass Learn              |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        63            | Sailboat Tack              |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        64            | Reverse Throttle           |          |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        65            | GPS Disable                |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        66            | Relay 5 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        67            | Relay 6 On/Off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        68            | STABILIZE mode             |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        69            | POSHOLD mode               |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        70            | ALTHOLD mode               |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        71            | FLOWHOLD mode              |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        72            | CIRCLE mode                |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        73            | DRIFT mode                 |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        74            | Sailboat Motor 3Pos Sw     |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        75            | Surface Tracking Up/Down   |          |         |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        76            | STANDBY mode               |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        77            | TAKEOFF mode               |          |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        207           | Mainsail (Sailboat)        |          |         |    X    |
```

Comment:
By the way, this is not actually an auxiliary function but rather a way to set the control for the mainsail to be done from an auxiliary switch.  Anyway, that distinction is perhaps too detailed for this page.. so it's fine.

Reply:
is the explanation below for it correct?